frontend/src/strings/crawl-workflows/infoText.ts

```diff
@@ -5,7 +5,7 @@ import { type FormState } from "@/utils/workflow";
 
 type Field = keyof FormState;
 
-const infoText: Partial<Record<Field, string | TemplateResult>> = {
+export const infoTextFor: Partial<Record<Field, string | TemplateResult>> = {
```

Comment:
This is better typed as: 
```ts
export const infoTextFor = {
  ...
} as const satisfies Partial<Record<Field, string | TemplateResult>>;
```
so that the keys it does have can be inferred and autocompleted, etc., and then it can be used as, e.g., `infoTextFor.exclusions` directly.